include/server/api/base_parameters_grammar.hpp

```diff
@@ -141,11 +141,18 @@ struct BaseParametersGrammar : boost::spirit::qi::grammar<Iterator, Signature>
                          add_hint, qi::_r1, qi::_1)] %
                          ';';
 
+        emit_hints_rule =
+            qi::lit("emit_hints=") >
+            qi::bool_[ph::bind(&engine::api::BaseParameters::emit_hints, qi::_r1) = qi::_1];
+
         bearings_rule =
             qi::lit("bearings=") >
             (-(qi::short_ > ',' > qi::short_))[ph::bind(add_bearing, qi::_r1, qi::_1)] % ';';
 
-        base_rule = radiuses_rule(qi::_r1) | hints_rule(qi::_r1) | bearings_rule(qi::_r1);
+        base_rule = radiuses_rule(qi::_r1)   //
```

Comment:
I have recently found `// clang-format off` `// clang-format on` and used here https://github.com/Project-OSRM/osrm-backend/blob/opening_hours/include/util/opening_hours.hpp#L302

Would it be ok to allow suppress clang format for qi rules  and use qi-usual formating like
```
    emit_hints_rule
        = qi::lit("emit_hints=")
        > qi::bool_[ph::bind(&engine::api::BaseParameters::emit_hints, qi::_r1) = qi::_1]
        ;
```
?